app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -1,3 +1,4 @@
+import cx from "classnames";
```

Comment:
I have a personal preference for more self explanatory names (even if longer). Can we just keep it `classNames` instead of `cx`?

Reply:
In most cases, I would definitely agree but `cx` is widely used for this particular package so I prefer to follow convention.
For example, see Line 1: https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/views/Card/Card.js

However if you still feel strongly about it, I am okay with `classNames`.

Reply:
Sure, we can keep it. I will make a note of common packages that have very specific names.